### PR TITLE
Modernize it all :tada:

### PR DESCRIPTION
**Pull request by LeoColomb**
Globally:
* Icon refresh for Chrome settings and displays
* `background.js` clean up and lighting
  * Using fetch instead of XHR
  * Using tabs.query instead of hard coded loop
* 🛩

I tried to keep my git log coherent, so please refer to it for details, then let me know what do you think about this!

### CODE REVIEW

**Comment by LeoColomb** [comment]
Oh, and be careful, your `editorconfig` is not coherent with the project, resulting in some indention mismatches.

**Comment by LeoColomb** [comment]
@Narno Up! 😸

**Comment by LeoColomb** [comment]
1. Yes, I've take this logo and export real sizes from the official `.ai` file.

**Comment by Narno** [comment]
* "indent_style = tab" HAAAAAAA, WTF I have done?! Thx.
* "use fetch API instead of old XHR": nice, I love that!

**Comment by Narno** [comment]
Another point: could you increment the manifest version number?

File: src/background.js [hunk 1]
```diff
@@ -157,24 +146,12 @@
...
-    switch (details.reason) {
-      case 'install':
-        chrome.runtime.openOptionsPage();
-        break;
-      default:
-        updateBadge();
-    }
+	chrome.runtime.onInstalled.addListener(details => {
+		if (details.reason === 'install') {
+			chrome.runtime.openOptionsPage();
```
Comment (LeoColomb) [inline src/background.js]: Well it didn't before (`default` case is not raised during installation), and it shouldn't: settings are not currently reviewed by the user, and an alarm is already created.

**Comment by LeoColomb** [comment]
> HAAAAAAA, WTF I have done?!

No problem 😉 

> fetch API instead of old XHR

Yes! And a little more efficient! 🔥 

> increment the manifest version number please

I can. But depending of how you manage versions, I would prefer you doing this... Let's say 2.0.0 (some breaks and major changes)?

**Comment by LeoColomb** [comment]
Done!

**Pull request merged**